### ISSUE TITLE
Split up 'Remote Sensor Closed Examples', add more Pigeon-integration examples

### ISSUE THREAD

**Issue by ozrien** [CrossTheRoadElectronics/Phoenix-Examples-LabVIEW#6]
The Remote Sensor Closed Loop examples demonstrates...
- Talon SRX / Victor SPX auxiliary closed looping on heading while closed looping pos/velocity/motionmagic
- Talon / Victor arbitrary feedforward for single-axis loops
- Talon / Victor aux heading closed loop via Motion Profile Arc
- Talon / Victor aux heading closed loop via Motion Profile Arc with feed forward
- Use of the remote sensor types (Pigeon IMU and remote Quadrature encoder)
- Use of sensor sum / sensor difference (for encoder-derived position and heading).

These examples should be broken out to several projects so that they can be found easier.

**Comment by jasondaming** [comment]
I agree that these should be split out.  I would also add that I think it should include example for the drivetrain (at least for motion profiling).  I believe you would call this "Independant axis motion profiling" (or just use tank drivetrain to find it more understandable for novices.
